### PR TITLE
Supplement the E2E test for skywalking-client-js

### PR DESCRIPTION
**Pull request by arugal**
Supplement the E2E test for skywalking-client-js

### CODE REVIEW

File: test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        test() {
+            throw {
+                msg: 'vue error',
+                status: 3000
+            }
+        }
+    },
+    created() {
+        this.click1();
+        this.test();
```
Comment (kezhenxu94) [inline test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js:56]: Here you throw 2 errors, why are there only one in the expected data?

Comment (Fine0830) [reply test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js:56]: When the first function throws an error, the second function is not executed.

Comment (Fine0830) [reply test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js:56]: Could you delete the second function? Probably it increases confusion.

File: .github/workflows/e2e.js.yaml [hunk 2]
```diff
@@ -60,3 +60,30 @@ jobs:
...
+      - name: Run E2E Test
+        if: env.SKIP_CI != 'true'
+        uses: ./.github/actions/e2e-test
+        with:
+          test_class: org.apache.skywalking.e2e.browser.BrowserWithClientJSE2E
+      - uses: actions/upload-artifact@v1
+        if: failure()
+        with:
+          name: logs
+          path: logs
```
[the remark this answers is not in the document]
Comment (arugal) [reply .github/workflows/e2e.js.yaml]: Updated, please recheck :)

File: test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+});
+
+// vue error
+new Vue({
+    data: {
+        name: "chen",
+        age: 18,
+        message: "--------------------------------",
+        src3: "https://pic.xiaohuochai.site/blog/chromePerformance2_error.png",
+        src4: "",
```
[the remark this answers is not in the document]
Comment (arugal) [reply test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js]: done

File: test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js (hunk 1 above)
Comment (arugal) [reply test/e2e/e2e-test/docker/browser/with-skywalking-client-js/index.js]: done

**Review by kezhenxu94 — DISMISSED** [review]
LGTM if tests passed

File: test/e2e/e2e-test/docker/browser/with-skywalking-client-js/Dockerfile.test-ui [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM node:10.23 AS builder
+
+## download and build skywalking client js
+ARG CLIENT_JS_COMMIT_HASH=04dd6b3b024f838b3b92002a370ce3b7f70638b4
```
Comment (wu-sheng) [inline test/e2e/e2e-test/docker/browser/with-skywalking-client-js/Dockerfile.test-ui:19]: Do you need to consider merging this first? And use the latest commit id?
https://github.com/apache/skywalking-client-js/pull/42

Comment (arugal) [reply test/e2e/e2e-test/docker/browser/with-skywalking-client-js/Dockerfile.test-ui]: done

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**